test/models/observation_test.rb

```diff
@@ -95,10 +125,23 @@ def test_specimens
     assert(observations(:detailed_unknown).specimens.length > 0)
   end
 
+  def test_observer_takes_general_email_questions_from_viewer
+    user_a = users(:rolf) # rolf takes general email questions
```

Comment:
It seems more intuitive if this was user_a = user_a_obs.user (obviously with the statements switched).